src/execution/index.ts

```
@@ -0,0 +1,79 @@
+    console.log('Executing with config:', config);
+    const files = fs.readdirSync(directory);
+    let output = '';
+
+    const reportGenerator = new ReportGenerator({
+      threshold: config.threshold || 0.8,
+      useSemanticScoring: config.useSemanticScoring || false,
+      outputFormat: (config.outputFormat as 'json' | 'markdown' | 'text') || 'json'
+    });
+
+    for (const file of files) {
+        const filePath = path.join(directory, file);
```

ESLint

[error] Delete `··`

(prettier/prettier)